NOnion/Utility/HiddenServicesUtility.fs

```diff
@@ -77,11 +77,15 @@ module HSUtility =
     let GetPublicKeyFromUrl(url: string) =
         //Remove .onion suffix and decode
         let keyBytesOpt =
-            url.Split('.') |> Seq.tryHead |> Option.map Base32Util.DecodeBase32
+            url.Split '.' |> Seq.tryHead |> Option.map Base32Util.DecodeBase32
```

Comment:
@aarani this kind of change should not be in this commit but in a refactoring commit

Reply:
@aarani pls don't mark things as resolved if they are not resolved

Reply:
We talked about this though.